components/molecule/photoUploader/src/index.js

```diff
@@ -387,6 +389,9 @@ MoleculePhotoUploader.propTypes = {
    */
   callbackUploadPhoto: PropTypes.func,
 
+  /** Component to render inside content space */
+  content: PropTypes.func,
```

Comment:
I've been thinking about that. Maybe content is too much ambiguous. Think about `fileChildren` or `childenFile` or something similar